data_model/master/device_types/HeatPump.xml

```diff
@@ -54,19 +54,19 @@ This notice and disclaimer must be included on all copies of this document.
 Connectivity Standards Alliance +
 508 Second Street, Suite 206 +
 Davis, CA 95616, USA
--->
-<deviceType xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="types types.xsd devicetype devicetype.xsd" id="0x0309" name="Heat Pump" revision="1">
-  <revisionHistory>
-    <revision revision="1" summary="Initial revision"/>
-  </revisionHistory>
-  <classification class="simple" scope="endpoint"/>
-  <conditions/>
-  <clusters>
-    <cluster id="0x0003" name="Identify" side="server">
-      <optionalConform/>
-    </cluster>
-    <cluster id="0x0301" name="Thermostat" side="client">
```

Comment:
This says 0x0301 because the _spec_ says 0x0301.  This needs to get fixed in the spec.  Then this file, which is auto-generated from the spec, will get regenerated with the right information.

Please file a spec issue on this, or create a PR against the spec.

Reply:
I wish I could. Unfortunately I don't have access to the specification.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10457 fixed this in the spec earlier today.

Reply:
Thanks